container-host-files/etc/scf/config/scripts/check_swapaccounting.sh

```diff
@@ -1,19 +1,33 @@
 #!/bin/sh
 
+set -x
+
 # This script checks if swap is available, and if yes, if swapaccounting is
 # enabled on the kernel.  It is an error to have swap without swapaccounting (as
 # that would lead to mis-tracking of memory usage).  If there is no swap,
 # however, not having swap accounting is safe; in that case we turn it off for
 # garden as well.
 
+# First a set of commands to show us what the following checks are seeing.
+
+grep Swap /proc/meminfo ;# Do we have swap, how much ?
+if [ -e /etc/default/grub ] ; then
+    grep CMDLINE /etc/default/grub ;# what does the boot loader say about swap/cgroup ?
```

Comment:
We should be looking at `/proc/cmdline` instead no? (Since that's the _currently running_ arguments)

Also: nit: use `-r` instead of `-e`, since it might exist but is not readable :D

Reply:
I see. Checking /proc/cmdline means I would not have to look for grub or other config file, because I query the kernel directly, and not the boot stuff ... That said, I could query both, in case it manages to not pick up what we think the config is.

Will change.

Reply:
Addressed with commit a280e36f66e6c86a4299200409588e618fbe7a6b